src/spec-node/featuresCLI/publish.ts

```diff
@@ -90,11 +90,45 @@ async function featuresPublish({
             process.exit(1);
         }
 
-        const thisResult = (publishResult?.digest && publishResult?.publishedVersions.length > 0) ? {
+        const ispublished = (publishResult?.digest && publishResult?.publishedVersions.length > 0);
+        let thisResult = ispublished ? {
             ...publishResult,
             version: f.version,
         } : {};
 
+        if (ispublished && f.legacyIds) {
+            output.write(`Processing legacyIds for '${f.id}'...`, LogLevel.Info);
+
+            let publishedLegacyIds: string[] = [];
+            for await (const legacyId of f.legacyIds) {
+                output.write(`Processing feature: ${legacyId}...`, LogLevel.Info);
```

Comment:
I think something like this would read better, given the output on line 100.

```suggestion
                output.write(`Processing legacyId '${legacyId}'...`, LogLevel.Info);
```